rust/gui-client/src-tauri/src/client/gui/system_tray_menu.rs

```diff
@@ -99,6 +101,54 @@ pub(crate) fn signed_out() -> SystemTrayMenu {
         .add_bottom_section(QUIT_TEXT_SIGNED_OUT)
 }
 
+pub(crate) fn debug() -> SystemTrayMenu {
```

Comment:
`cargo run -p firezone-gui-client -- debug tray-menu`

Should show fake resources 1 through 6 inclusive